### PR TITLE
Fix Input Field annotations handling

### PR DESCRIPTION
**Pull request by bartv2**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | yes
| Fixed tickets | #... 
| License       | MIT

- Skip properties without a Field annotation
- Enable field type guessing for Input annotation

### CODE REVIEW

File: src/Config/Parser/AnnotationParser.php [hunk 1]
```diff
@@ -622,16 +622,28 @@ private static function getGraphQLInputFieldsFromAnnotations(GraphClass $graphCl
...
+
             // Ignore field with resolver when the type is an Input
             if (isset($fieldAnnotation->resolve)) {
                 return [];
             }
 
             $fieldName = $reflector->getName();
-            $fieldType = $fieldAnnotation->type;
+            $fieldType = $fieldAnnotation->type ?? null;
+            if (!$fieldType) {
```
Comment (Vincz) [inline src/Config/Parser/AnnotationParser.php:640]:
```suggestion
            if (!isset($fieldAnnotation->type)) {
```

**Comment by Vincz** [comment]
Good job @bartv2 !
Can you do the same in `getTypeFieldConfigurationFromReflector` for regular GraphQL types ? 
Trying to get the type from type hint first ?

**Comment by bartv2** [comment]
Something like this?

**Comment by Vincz** [comment]
@bartv2 Yes ! Looks good ! Just run a little `composer fix-cs` and update the doc about auto-guessing and we are good to go :-)

**Pull request merged**